src/matchers/toBeHexadecimal/predicate.js

```diff
@@ -0,0 +1,5 @@
+export default expected => {
+  const re1 = RegExp(/^#\b[a-f0-9]{6}\b/gi);
+  const re2 = RegExp(/^#\b[a-f0-9]{3}\b/gi);
+  return re1.test(expected) || re2.test(expected);
```

Comment:
is it worth naming these a bit more explicitly, e.g.
shortRegex
longRedex
I only ask because i read the regex 3 times before i spotted the 6 and 3 🌜

Reply:
Agreed. This was lazy on my part.